packages/editor-ui/src/views/SettingsApiView.vue

```diff
@@ -105,6 +106,10 @@ export default mixins(
 		},
 		async createApiKey() {
 			this.loading = true;
+			this.$telemetry.track('User clicked create API key button', {
+				user_id: this.$store.getters['users/currentUserId'],
+				instance_id: this.$store.getters.instanceId,
```

Comment:
applies here and below... you don't need user id or instance id. they are included by default

Reply:
Thank you. Removed!